src/inverseRobot.js

```diff
@@ -24,7 +24,23 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robot2 = {};
+
+  for (let i = 0; i < Object.keys(robot).length; i++) {
```

Comment:
DRY 
`Object.keys(robot)` - move to variable
`Object.values(robot)` - move to variable